.idea/runConfigurations.xml

```diff
@@ -1,10 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<project version="4">
-  <component name="RunConfigurationProducerService">
-    <option name="ignoredProducers">
-      <set>
-        <option value="com.android.tools.idea.compose.preview.runconfiguration.ComposePreviewRunConfigurationProducer" />
-      </set>
-    </option>
-  </component>
-</project>
```

Comment:
(not saying you shouldn't since I am not sure of why we need it in the first place) but why is this deleted?

Reply:
> (not saying you shouldn't since I am not sure of why we need it in the first place) but why is this deleted?

I really don't know. This gets deleted when I run the app I think.

Reply:
Yes I see, this is happening also on PR #220. We should ask @rovati if this is needed since he introduced it in the codebase.

Reply:
It can be safely removed, I committed it by mistake a while ago, I don't know why it gets generated for me when building the code